frontend/src/pages/LogsPage/LogsTable/LogsTable.tsx

```diff
@@ -119,56 +119,48 @@ const LogsTableInner = ({
 	const queryTerms = parseLogsQuery(query)
 	const [expanded, setExpanded] = useState<ExpandedState>({})
 
-	const columns = React.useMemo<ColumnDef<LogEdge>[]>(
-		() => [
-			{
-				accessorKey: 'node.timestamp',
-				cell: ({ row, getValue }) => (
-					<Box
-						flexShrink={0}
-						flexDirection="row"
-						display="flex"
-						alignItems="flex-start"
-						gap="6"
-					>
-						{row.getCanExpand() && (
-							<Box
-								display="flex"
-								alignItems="flex-start"
-								cssClass={styles.expandIcon}
-							>
-								{row.getIsExpanded() ? (
-									<IconExpanded />
-								) : (
-									<IconCollapsed />
-								)}
-							</Box>
-						)}
-						<LogTimestamp timestamp={getValue() as string} />
-					</Box>
-				),
-			},
-			{
-				accessorKey: 'node.level',
-				cell: ({ getValue }) => (
-					<LogLevel level={getValue() as LogLevelType} />
```

Comment:
Same here:
<img width="913" alt="Screenshot 2023-06-26 at 10 06 47 PM" src="https://github.com/highlight/highlight/assets/58678/fdf05698-4621-4dec-a1b5-0ddaa95942db">

Note `LogLevelType` was an alias for `LogLevel`